Lib/test/test_httpservers.py

```diff
@@ -1535,6 +1537,197 @@ def test_server_test_ipv4(self, _):
             server.test(ServerClass=mock_server, bind=bind)
             self.assertEqual(mock_server.address_family, socket.AF_INET)
 
+@mock.patch('http.server.test')
+class CommandLineTestCase(unittest.TestCase):
+    def setUp(self):
+        self.default_port = 8000
+        self.default_bind = None
+        self.default_protocol = 'HTTP/1.0'
+        self.default_handler = SimpleHTTPRequestHandler
+        self.default_server = unittest.mock.ANY
+        self.tls_cert = certdata_file('ssl_cert.pem')
+        self.tls_key = certdata_file('ssl_key.pem')
+        self.tls_password = 'somepass'
+        tls_password_file_object = \
+            tempfile.NamedTemporaryFile(mode='w+', delete=False)
+        tls_password_file_object.write(self.tls_password)
+        self.tls_password_file = tls_password_file_object.name
+        tls_password_file_object.close()
+        self.args = {
+            'HandlerClass': self.default_handler,
+            'ServerClass': self.default_server,
+            'protocol': self.default_protocol,
+            'port': self.default_port,
+            'bind': self.default_bind,
+            'tls_cert': None,
+            'tls_key': None,
+            'tls_password': None,
+        }
+        return super().setUp()
+
+    def tearDown(self):
+        if os.path.exists(self.tls_password_file):
+            os.remove(self.tls_password_file)
+        return super().tearDown()
```

Comment:
we can also make this code more concise:
```suggestion
    default_port = 8000
    default_bind = None
    default_protocol = 'HTTP/1.0'
    default_handler = SimpleHTTPRequestHandler
    default_server = unittest.mock.ANY
    tls_cert = certdata_file('ssl_cert.pem')
    tls_key = certdata_file('ssl_key.pem')
    tls_password = 'somepass'
    args = {
        'HandlerClass': default_handler,
        'ServerClass': default_server,
        'protocol': default_protocol,
        'port': default_port,
        'bind': default_bind,
        'tls_cert': None,
        'tls_key': None,
        'tls_password': None,
    }

    def setUp(self):
        self.tls_password_file = tempfile.mktemp()
        with open(self.tls_password_file, 'wb') as f:
            f.write(self.tls_password.encode())
        self.addCleanup(os_helper.unlink, self.tls_password_file)
```

Reply:
This is indeed better.

Reply:
@ggqlq seems like you missed this suggestion :)

Reply:
@donBarbos  My bad! I made the change in the newest commit.

Reply:
thanks, but i'm not sure that we need to call `super().setUp()`, this method literally does nothing

Reply:
It's always better to call it because it prevents future mistakes if we were to change the mro.